networkx/algorithms/tests/test_tournament.py

```diff
@@ -1,132 +1,157 @@
 """Unit tests for the :mod:`networkx.algorithms.tournament` module."""
 from itertools import combinations
-
-
+import pytest
 from networkx import DiGraph
 from networkx.algorithms.tournament import is_reachable
 from networkx.algorithms.tournament import is_strongly_connected
 from networkx.algorithms.tournament import is_tournament
 from networkx.algorithms.tournament import random_tournament
 from networkx.algorithms.tournament import hamiltonian_path
+from networkx.algorithms.tournament import score_sequence
+from networkx.algorithms.tournament import tournament_matrix
+from networkx.algorithms.tournament import index_satisfying
+
+np = pytest.importorskip("numpy")
+npt = pytest.importorskip("numpy.testing")
+
+
+def test_condition_not_satisfied():
+    condition = lambda x: x > 0
+    assert index_satisfying([0], condition) == 1
```

Comment:
This one might benefit from a little more description. Without context for the expected behavior of `index_satisfying` when no index meeting the condition is found, this test is hard to understand. Maybe something like:

```suggestion
def test_condition_not_satisfied():
    condition = lambda x: x > 0
    iter_in = [0]
    assert index_satisfying(iter_in, condition) == len(iter_in)
```

Alternatively you can leave the test as-is and just mention the expected behavior in a brief docstring:

```suggestion
def test_condition_not_satisfied():
    """index_satisfying returns length of iterable when condition not met."""
    condition = lambda x: x > 0
    assert index_satisfying([0], condition) == 1
```